lib/checkpoint.js

```diff
@@ -3,7 +3,7 @@ const figures = require('figures')
 const util = require('util')
 
 module.exports = function (argv, msg, args, figure) {
-  const defaultFigure = args.dryRun ? chalk.yellow(figures.tick) : chalk.green(figures.tick)
+  const defaultFigure = argv.dryRun ? chalk.yellow(figures.tick) : chalk.green(figures.tick)
```

Comment:
Just wanted to highlight this change, `args.dryRun` seems to have been a typo.
Looking at the usage of `args` the expected is an array and not an object.

https://github.com/conventional-changelog/standard-version/blob/4383381f85b75cb35ca9e779f1c9b2505196aa10/lib/checkpoint.js#L8